File: Sorting/gcd.c
#include<stdio.h>
int gcd(int a,int b);  //15 9
int main(){
    int a=4;
    int b=2;
    printf("%d",gcd(a,b));

}
int gcd(int a,int b){
    if(b==0){
        return a;
    }
    return gcd(b,a%b);
}

File: Recursion/learn/Gcd.java
public class Gcd {
    public static int gcd(int x,int y){ //24,15
        if(y==0){
            return x;
        }
        return gcd(y,x%y);

    }
    public static void main(String args[]){
        System.out.println(gcd(24,15));

    }
    
}
